docs/api-reference/extensions/v1beta1/definitions.md

```diff
@@ -31,7 +31,7 @@ Documentation for other releases can be found at
 
 <!-- END MUNGE: UNVERSIONED_WARNING -->
 <!-- needed for gh-pages to render html files when imported -->
-{% include extensions-v1beta1-definitions.html %}
+{% include <REPLACE-WITH-RELEASE-VERSION>/extensions-v1beta1-definitions.html %}
```

Comment:
Note that this will set it to v1.1.0.
It seems that you want it to be v1.1 as per https://github.com/kubernetes/kubernetes/pull/16213